docs/howto/testing.rst

```diff
@@ -0,0 +1,181 @@
+################################
+Running and debugging Cylc Tests
+################################
+
+================
+How to run tests
+================
+
+
+All tests are in the folder ``cylc-flow/tests``. Instructions
+here, unless stated otherwise assume that you are working from
+your cylc-flow repo dir.
+
```

Comment:
Worth putting a comment in here about flow-tests.rc file?

Reply:
Agreed!

Reply:
Do we have an example of flow tests.rc?